RELEASE_PROCESS.md

```diff
@@ -1,4 +1,2 @@
-See the [consensus release process file in the `ouroboros-network`
-repository](https://github.com/input-output-hk/ouroboros-consensus/blob/master/docs/ReleaseProcess.md).
-Note that we do not do any bundling of packages: each package has its own
-changelog, and each can be released separately.
\ No newline at end of file
+See the [consensus release process file in the `ouroboros-consensus`
```

Comment:
We could consider dropping this if it's overkill, like we did in our other repos.

Reply:
Done